specification/common-types/resource-management/v1/privatelinks.json

```diff
@@ -22,6 +22,9 @@
           "$ref": "#/definitions/PrivateEndpointConnectionProperties",
           "x-ms-client-flatten": true,
           "description": "Resource properties."
+        },
+        "systemData": {
+          "$ref": "types.json#/definitions/systemData"
```

Comment:
Since this is common definition. please add `description` for this property.

Reply:
Description added. thanks

Reply:
@moria97 Why do you add the property in the common-types? This file is referenced by many RPs. You'd better add it in your own swagger.

Reply:
Actually we only need to return this data model defined in privatelinks.json directly. However, it will fails the LintDiff check with R4009 error which says response of GET, PUT should have a systemData property. So I choose to add the property in common-types that other people can avoid this error.
Now I have updated these by adding a wrapper in my own swagger file which combines original definition with additional systemData field. Maybe we can have a call to discuss this issue.

Reply:
systemData property is only required for new apiVersion(later than 2020-05-01 IIRC). So currently it's better to add it in your own swagger. @jianyexi correct me if wrong.